plugins/events/create.js

```diff
@@ -76,6 +77,10 @@ module.exports = () => ({
                     payload.parentBuildId = parentBuildId;
                 }
 
+                if (parentBuilds || parentBuildsInfo) {
```

Comment:
based on previous comment, can't his just be passed `parentBuilds` here ? instead of introducing a `parentBuildsInfo`, can't we just have `parentBuilds` ?

Reply:
it can be passed into the event start config or gotten from a build